core/css.js

```diff
@@ -17,1007 +17,904 @@
-    'fill: none;',
-    'pointer-events: all;',
-  '}',
-
-  '.blocklyFlyoutButtonBackground {',
-      'stroke: #c6c6c6;',
-  '}',
-
-  '.blocklyFlyoutButton .blocklyText {',
-    'fill: $colour_textFieldText;',
-  '}',
-
-  '.blocklyFlyoutButtonShadow {',
-    'fill: transparent;',
-  '}',
-
-  '.blocklyFlyoutButton:hover {',
-    'fill: white;',
-    'cursor: pointer;',
-  '}',
-
-  '.blocklyFlyoutLabel {',
-    'cursor: default;',
-  '}',
-
-  '.blocklyFlyoutLabelBackground {',
-    'opacity: 0;',
-  '}',
-
-  '.blocklyTouchTargetBackground {',
-    'fill: transparent;',
-    'cursor: pointer;',
-  '}',
-
-  '.blocklyFlyoutLabelText {',
-    'font-family: "Helvetica Neue", Helvetica, sans-serif;',
-    'font-size: 14pt;',
-    'fill: #575E75;',
-    'font-weight: bold;',
-  '}',
+    cursor: url("<<<PATH>>>/handclosed.cur"), auto;
+    cursor: grabbing;
+    cursor: -webkit-grabbing;
+    cursor: -moz-grabbing;
+  }
+
+  .blocklyDragging.blocklyDraggingDelete {
+    cursor: url("<<<PATH>>>/handdelete.cur"), auto;
+  }
+
+  .blocklyDragging.blocklyDraggingMouseThrough {
+    pointer-events: none;
+  }
+
+  .blocklyToolboxDelete {
+    cursor: url("<<<PATH>>>/handdelete.cur"), auto;
+  }
+
+  .blocklyToolboxGrab {
+    cursor: url("<<<PATH>>>/handclosed.cur"), auto;
+    cursor: grabbing;
+    cursor: -webkit-grabbing;
+  }
+
+  .blocklyDragging>.blocklyPath,
+  .blocklyDragging>.blocklyPathLight {
+    fill-opacity: 1.0;
+    stroke-opacity: 1.0;
+  }
+
+  .blocklyDragging>.blocklyPath {
+  }
+
+  .blocklyDisabled>.blocklyPath {
+    fill-opacity: .5;
+    stroke-opacity: .5;
+  }
+
+  .blocklyInsertionMarker>.blocklyPath {
+    stroke: none;
+  }
+
+  .blocklyText {
+    fill: var(--colour-text);
+    font-family: "Helvetica Neue", Helvetica, sans-serif;
+    font-size: 12pt;
+    font-weight: 500;
+  }
+
+  .blocklyTextTruncated {
+    font-size: 11pt;
+  }
+
+  .blocklyNonEditableText>text {
+    pointer-events: none;
+  }
+  .blocklyNonEditableText>text,
+  .blocklyEditableText>text {
+    fill: var(--colour-textFieldText);
+  }
+
+  .blocklyEditableText>.blocklyEditableLabel {
+    fill: #fff;
+  }
+
+  .blocklyDropdownText {
+    fill: $colour_text !important;
+  }
+
+  .blocklyBubbleText {
+    fill: var(--colour-textFieldText);
+  }
+  .blocklyFlyout {
+    position: absolute;
+    z-index: 20;
+  }
+  .blocklyFlyoutButton {
+    fill: none;
+    pointer-events: all;
+  }
+
+  .blocklyFlyoutButtonBackground {
+      stroke: #c6c6c6;
+  }
+
+  .blocklyFlyoutButton .blocklyText {
+    fill: var(--colour-textFieldText);
+  }
+
+  .blocklyFlyoutButtonShadow {
+    fill: transparent;
+  }
+
+  .blocklyFlyoutButton:hover {
+    fill: white;
+    cursor: pointer;
+  }
+
+  .blocklyFlyoutLabel {
+    cursor: default;
+  }
+
+  .blocklyFlyoutLabelBackground {
+    opacity: 0;
+  }
+
+  .blocklyTouchTargetBackground {
+    fill: transparent;
+    cursor: pointer;
+  }
+
+  .zelos-renderer.scratch-theme .blocklyFlyoutLabelText {
```

Comment:
Ugh I hate that the renderers do this, I think it's just actually wrong.

(non-actionable, just me complaining)